client/src/components/csv/CSVReview.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import {Container, Label, Item} from 'semantic-ui-react';
+import { ItemComplete } from "../item/ItemEdit";
+
+interface ReviewCardProps {
+    item: ItemComplete
+}
+
+interface ReviewProps {
+    inventory: ItemComplete[]
+}
+
+const ReviewSetup = (props: ReviewProps) => {
+    const { inventory } = props;
+
+    return (
+        <Container>
+            <Item.Group>
+                {inventory.map(item => <ReviewCard key={item.name} item={item}/>)}
+            </Item.Group>
+        </Container>
+    );
+}
+
+const ReviewCard = (props: ReviewCardProps) => {
+    const { item } = props;
+    const { name, description, totalQty, maxReqQty, 
+            imageUrl, category, price, owner, 
+            requireApproval, returnRequired, hidden} = item;
+    return (
+        <Item>
+            <Item.Image size='tiny' src={imageUrl}/>
+            <Item.Content>
+                <Item.Header as='h4'>{name}</Item.Header>
+                <Item.Meta>{maxReqQty} requests limit | {totalQty} available, Owner: {owner}, Unit Cost: {price}</Item.Meta>
```

Comment:
>{maxReqQty} requests limit
The proper terminology here is (for consistency with inventory list on home page) "Request up to 3 at a time."  Conveniently, this also avoids the need to handle plurals, although "Request up to 1 at a time" sounds slightly awkward to me.

Also, please add a $ dollar sign before the value of Unit Cost